internal/service/rekognition/stream_processor.go

```diff
@@ -267,22 +264,17 @@ func (r *resourceStreamProcessor) Schema(ctx context.Context, req resource.Schem
 							},
 							NestedObject: schema.NestedBlockObject{
 								CustomType: fwtypes.NewObjectTypeOf[polygonModel](ctx),
-								Validators: []validator.Object{
-									objectvalidator.AlsoRequires(
-										path.MatchRelative().AtName("x"),
-										path.MatchRelative().AtName("y"),
-									)},
 								Attributes: map[string]schema.Attribute{
 									"x": schema.Float64Attribute{
 										Description: "The value of the X coordinate for a point on a Polygon.",
-										Optional:    true,
+										Required:    true,
```

Comment:
Should these remain optional? 
https://docs.aws.amazon.com/rekognition/latest/APIReference/API_Point.html#rekognition-Type-Point-X